src/TableGateway/AbstractTableGateway.php

```diff
@@ -25,25 +25,25 @@
 /**
  *
  * @property AdapterInterface $adapter
- * @property int $lastInsertValue
- * @property string $table
+ * @property int              $lastInsertValue
+ * @property string           $table
  */
 abstract class AbstractTableGateway implements TableGatewayInterface
 {
     /**
      * @var bool
      */
-    protected $isInitialized = false;
+    protected $isInitialized = FALSE;
```

Comment:
@AdamiecRadek i think this shouldn't be changed. Also the other occurs of `null` and `false`